src/pages/UpdateServers.js

```diff
@@ -93,8 +102,62 @@ class UpdateServers extends BaseUpdateWizardPage {
           };
         });
       });
+    this.checkMonasca();
+  }
+
+  /**
+   * checks to see if Monasca is installed, and if it is, triggers a call to the status
+   * of each server in the model
+   */
+  checkMonasca() {
+    if(this.state.monasca === undefined) {
+      fetchJson('/api/v1/clm/monasca/is_installed')
+        .then(responseData => {
+          if(responseData.installed) {
+            getInternalModel()
+              .then((yml) => {
+                this.setState({internalModel: yml, monasca: true}, () => this.getServerStatuses());
+              }).catch((error) => {
+                console.log('error getting internal model:' + error);// eslint-disable-line no-console
```

Comment:
If this line is executed, then the monasca state variable will remain undefined, despite the fact that we know that it is installed. That smells wrong.